components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,107 @@
+const common = require("../../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      propDefinition: [reddit, "subreddit"],
+    },
+    subreddit_id36: {
+      type: "string",
+      label: "ID36 of Subreddit",
+      description:
+        "Pipedream will emit a new event when a comment is posted to the subreddit pointed by this ID36.",
+      optional: false,
+    },
+    context: {
+      type: "integer",
+      label: "Context (0 to 8)",
+      description:
+        "If set to 0, it will include the new comment in the emitted event. Otherwise, it specifies how many parents of the new comment will be included in said event.",
+      default: 0,
+      optional: false,
```

Comment:
Having it mandatory and with a default value doesn't make much sense. A better approach would be this:
```js
module.exports = {
  props: {
    context: {
      // Other attributes here...
      default: 0,
      min: 0,
      max: 8,
    },
  },
};
```

Reply:
Interesting, was much conscious of the min/max values although they are in the component docs. thanks for noting.